src/sentry/static/sentry/app/views/releasesV2/list/releaseHealth.tsx

```diff
@@ -34,12 +34,27 @@ type Props = {
   release: Release;
   orgSlug: string;
   location: Location;
+  selection: GlobalSelection;
 };
 
-const ReleaseHealth = ({release, orgSlug, location}: Props) => {
+const ReleaseHealth = ({release, orgSlug, location, selection}: Props) => {
   const activeStatsPeriod = (location.query.healthStatsPeriod || '24h') as StatsPeriod;
   const activeStatsSubject = (location.query.healthStat || 'sessions') as StatsSubject;
 
+  // sort health rows inside release card alphabetically by project name,
+  // but put the ones with project selected in global header to top
+  const selectedProjects: ReleaseProject[] = [];
+  const otherProjects: ReleaseProject[] = [];
+  release.projects
+    .sort((a, b) => a.slug.localeCompare(b.slug))
+    .forEach(p => {
```

Comment:
Would [lodash/partition](https://lodash.com/docs/4.17.15#partition) make this simpler?

Reply:
TIL, thanks! Much better now.